documentation/source/newsfragments/1471.bugfix.rst

```diff
@@ -0,0 +1 @@
+fix verilator.cpp for writing coverage files when calls to vl_finish end tests
```

Comment:
```suggestion
Fixed Verilator not writing coverage files in some cases.
```

Also update this file name to `1478.bugfix.rst`

Reply:
fixed in recent push.